roles/confluent.kafka_broker/defaults/main.yml

```diff
@@ -22,7 +23,7 @@ kafka_broker_service_overrides:
   User: "{{ kafka_broker_user if kafka_broker_user != kafka_broker_default_user else '' }}"
   Group: "{{ kafka_broker_group if kafka_broker_group != kafka_broker_default_group else '' }}"
 kafka_broker_service_environment_overrides:
```

Comment:
If you are looking to be able to override the kafka_broker_heap_opts, you dont need to put it in a new variable you can jsut put this in your inventory file:

```
kafka_broker_service_environment_overrides:
  KAFKA_HEAP_OPTS: "-Xmx2g"
```
I suppose long term we should move away from hash merging... but it does work well :)

Reply:
Agree. Just had not figured out about merging, by that time